config/Dockerfiles/fedoraci-runner/rpmbuild/koji_build_pr.sh

```diff
@@ -61,15 +61,30 @@ koji ${KOJI_PARAMS} build --wait --arch-override=x86_64 --scratch ${branch} ${fe
 RPMBUILD_RC=$?
 if [ "$RPMBUILD_RC" != 0 ]; then
      echo "status=FAIL" >> ${LOGDIR}/job.props
-     echo -e "ERROR: KOJI BUILD\nSTATUS: $MOCKBUILD_RC"
+     echo -e "ERROR: KOJI BUILD\nSTATUS: $RPMBUILD_RC"
      exit 1
 fi
-echo "status=SUCCESS" >> ${LOGDIR}/job.props
+
 popd
 
 SCRATCHID=$(cat ${LOGDIR}/kojioutput.txt | awk '/Created task:/ { print $3 }')
 echo "koji_task_id=${SCRATCHID}" >> ${LOGDIR}/job.props
 
+# Make sure koji build finished
+# https://pagure.io/fedora-ci/general/issue/76
+while koji taskinfo ${SCRATCHID} | grep "State: open"; do
```

Comment:
this (taskinfo) could have a temporary error that leads to grep failure. also, the status could be "free". should this grep the other way around (i.e. continue to loop if it cannot grep closed|failed|cancelled?